relay-metrics/src/view.rs

```diff
@@ -96,12 +96,12 @@ impl<'a> BucketsView<'a> {
     }
 
     /// Iterator over all buckets in the view.
-    pub fn iter(&self) -> impl Iterator<Item = BucketView<'a>> {
+    pub fn iter(&self) -> BucketsViewIter<'a> {
```

Comment:
Since the iterator types are now public, we can name them in these methods here, too. This is also in accordance with our style guide.